packages/cli/src/ui/components/Header.tsx

```diff
@@ -28,11 +28,14 @@ export const Header: React.FC<HeaderProps> = ({
   const widthOfLongLogo = getAsciiArtWidth(longAsciiLogo);
   const widthOfShortLogo = getAsciiArtWidth(shortAsciiLogo);
 
+  // Small hysteresis to reduce toggling between logo sizes around width thresholds.
```

Comment:
nit: move this to a separate tiny pr as it is unrelated and not sure adding this is needed particularly given the header is static so only refreshed when we force a refresh.